.travis.yml

```diff
@@ -5,10 +5,15 @@ elixir:
   - 1.8.2
   - 1.9.1
 
+cache:
+  directories:
+    - priv/plts
+
 jobs:
   include:
     - stage: lint
       elixir: 1.9.1
       script:
         - mix format --check-formatted
         - mix credo
+        - MIX_ENV=dev travis_wait 20 mix dialyzer --plt
```

Comment:
Maybe this is worth running as a separate job? we could call it `dyalizer`

Reply:
good call